source/configurator/CMakeLists.txt

```diff
@@ -9,7 +9,10 @@ project(RoR_Configurator)
 
 if (WIN32)
     set(OS_LIBS "${PThread_LIBRARIES};dinput8.lib;dxguid.lib")
-    set(OS_SOURCE "icon.rc")
+    # clang-cl doesn't support resource files

```

Comment:
Tabs -> Spaces

Reply:
edited before changing the editor's settings to use spaces,it should be fixed in the next commit